framework/src/modules/chain/bft/errors.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const { FrameworkError } = require('../../../errors');
+
+class BFTError extends FrameworkError {}
+
+class BFTChainDisjointError extends BFTError {
+	constructor() {
+		super(
+			'Violates disjointness condition. If delegate forged a block of higher height earlier and later the block with lower height'
```

Comment:
```suggestion
			'Violation of disjointness condition. If delegate forged a block of higher height earlier and later the block with lower height'
```

To keep wording consistency